qcodes/tests/dataset/test_dataset_basic.py

```diff
@@ -51,7 +57,7 @@ def test_tables_exist(empty_temp_db):
     print(qc.config["core"]["db_location"])
     conn = connect(qc.config["core"]["db_location"], qc.config["core"]["db_debug"])
     cursor = conn.execute("select sql from sqlite_master where type = 'table'")
-    expected_tables = ['experiments', 'runs', 'layouts', 'dependencies']
+    expected_tables = ['experiments', 'runs', 'layouts', 'dependencies', 'uuids']
```

Comment:
what is the `uuids` table?

Reply:
An obsolete idea: a separate tables for the guids (called uuids back then). This should not still be in the PR, thanks for catching it.

Reply:
ok. i wonder why this hasn't been noticed when CI runs there tests...... could you have look? perhaps the test is not working?

Reply:
So the explanation was that the `uuids` table was indeed no longer in the database, but the test did not catch that, since it did a 
```
    for row, expected_table in zip(cursor, expected_tables):
        assert expected_table in row['sql']
```
so if we had *more* expected tables than tables in the DB (i.e. rows in the cursor), the test wouldn't notice. That has been corrected now.